### DISCUSSION TITLE
Configure sub-taglines in website homepage

### DISCUSSION THREAD

**Discussion by tuhaihe** [facebook/docusaurus#9207] [Q&A] [answered]
Hello! I have successfully deployed Docusaurus for one project, and I have a question about configuring `sub-taglines`. Our project has a main slogan or tagline, as well as one sub-tagline. I have set the main tagline field `tagline:` in the file `docusaurus.config.js`.

Can someone please provide instructions on how to set up the sub-tagline on the second line? I would be grateful for any help you can offer. Thank you!

**Comment by slorber** [comment, marked as answer]
We don't have a concept of "sub-tagline", where does this come from?

Even the concept of "tagline" is not that useful IMHO, since you can customize everything by implementing your own React landing page.

**Comment by tuhaihe** [comment]
Thanks for your reply @slorber! 
Perhaps I need to ask for help from our company's front-end team.
